src/main/java/com/gojek/beast/config/BQConfig.java

```diff
@@ -45,4 +45,8 @@
     @DefaultValue("-1")
     @Key("BQ_CLIENT_CONNECT_TIMEOUT")
     String getBqClientConnectTimeout();
+
+    @DefaultValue("-1")
+    @Key("BQ_TABLE_PARTITION_EXPIRY")
```

Comment:
use BQ_TABLE_PARTITION_EXPIRY_IN_MS, so that it is explicit

Reply:
`BQ_TABLE_PARTITION_EXPIRY_MILLIS` is better since MS can mean microseconds also.